### PR TITLE
Add Release Notes for iOS Release 1.57

### LINKED ISSUE #8029: Release Notes for iOS Release 1.57
### Web3

- Added search bar to “Select a Token to Send” view. ([#7768](https://github.com/brave/brave-ios/issues/7768))
- Added “Group by” filter to Portfolio tab in Brave Wallet. ([#7765](https://github.com/brave/brave-ios/issues/7765))
- Added NFT filters and display settings in the NFT tab for Brave Wallet. ([#7699](https://github.com/brave/brave-ios/issues/7699))
- Added new UI for the Brave Wallet onboarding flow. ([#7619](https://github.com/brave/brave-ios/issues/7619))
- Added handling of empty state(s) for Brave Wallet Portfolio filters. ([#7618](https://github.com/brave/brave-ios/issues/7618))
- Added filters and display settings for Portfolio tab in Brave Wallet. ([#7585](https://github.com/brave/brave-ios/issues/7585))
- Enabled Sardine On-ramp purchase option for all wallet users. ([#7608](https://github.com/brave/brave-ios/issues/7608))
- Fixed crash when switching to Polygon Network in Brave Wallet “Swap” view. ([#7907](https://github.com/brave/brave-ios/issues/7907))
- Fixed “Select Account” panel to always show checkmark for the selected account. ([#7823](https://github.com/brave/brave-ios/issues/7823))

### General

- Implemented query parameter filtering when navigating between websites. ([#7861](https://github.com/brave/brave-ios/issues/7861))
- Added biometric authentication for Private Mode. ([#7810](https://github.com/brave/brave-ios/issues/7810))
- Added “Private Tabs” settings section in main settings menu. ([#7855](https://github.com/brave/brave-ios/issues/7855))
- Added option to keep private tabs when the user exits the browser. ([#3183](https://github.com/brave/brave-ios/issues/3183))
- Added support for Multi-Window on iPad. ([#7574](https://github.com/brave/brave-ios/issues/7574))
- Added haptic feedback for voice search when entry is successful. ([#7798](https://github.com/brave/brave-ios/issues/7798))
- Added Privacy-Preserving Product Analytics (P3A) reporting for New Tab Page Sponsored Images. ([#6806](https://github.com/brave/brave-ios/issues/6806))
- Added support for compiling blocklists from raw text filter lists. ([#5975](https://github.com/brave/brave-ios/issues/5975))
- Added cookie consent notice popup as opt-out. ([#7792](https://github.com/brave/brave-ios/issues/7792))
- Added “View Recently Closed Tabs” to the tab tray context menu. ([#7752](https://github.com/brave/brave-ios/issues/7752))
- Added network level standard blocking for cosmetic filtering. ([#7539](https://github.com/brave/brave-ios/issues/7539))
- Updated the strings in the “restore your tabs” message (after experiencing a crash). ([#7695](https://github.com/brave/brave-ios/issues/7695))
- Updated the URL bar context menu to show by long pressing on the search field as well the refresh icon in the URL bar. ([#7642](https://github.com/brave/brave-ios/issues/7642))
- Updated New Tab Page favicon to display Brave icon instead of generic planet icon. ([#5357](https://github.com/brave/brave-ios/issues/5357))
- Updated “Block Cookie Consent Notices” to be enabled by default. ([#7882](https://github.com/brave/brave-ios/issues/7882))
- Updated logic for the Default Browser prompt on New Tab Page. ([#7860](https://github.com/brave/brave-ios/issues/7860))
- Updated Sync Chain authentication logic to support Biometrics when enabling password sync or adding a new device. ([#7849](https://github.com/brave/brave-ios/issues/7849))
- Updated that VPN Support Contact Form to append the issue category to the email subject field as well as body. ([#7828](https://github.com/brave/brave-ios/issues/7828))
- Fixed issue with “Block Cookie Consent Notices” toggle switch being enabled when upgrading browser version. ([#7920](https://github.com/brave/brave-ios/issues/7920))
- Fixed issue with Google OAuth window not auto-closing after successfully authenticating. ([#6966](https://github.com/brave/brave-ios/issues/6966))
- Fixed URLs for i.redd.it posts on subreddits from forming incorrectly when using [old.reddit.com](http://old.reddit.com/) website redirect. ([#7873](https://github.com/brave/brave-ios/issues/7873))
- Fixed issue allowing users to bypass the Browser Lock window using the “Cancel” button. ([#7893](https://github.com/brave/brave-ios/issues/7893))
- Fixed “Brave can now be set as your default browser…” prompt to not be shown 14 days after installation. ([#7977](https://github.com/brave/brave-ios/issues/7977))
- Fixed crash when migrating with very many tabs open. ([#7857](https://github.com/brave/brave-ios/issues/7857))
- Fixed crash when viewing an advertisement in the Brave News feed after joining Brave Rewards. ([#7837](https://github.com/brave/brave-ios/issues/7837))
- Fixed Tabs Bar to update properly when opening bulk bookmarks. ([#7836](https://github.com/brave/brave-ios/issues/7836))
- Fixed restored but unopened tabs from being added to “Recently Closed Tabs” page. ([#7832](https://github.com/brave/brave-ios/issues/7832))
- Fixed tabs to close properly with window.close(). ([#7796](https://github.com/brave/brave-ios/issues/7796))
- Fixed crash in tab view when tapping “Private” flag while actively dragging a tab. ([#7737](https://github.com/brave/brave-ios/issues/7737))
- Fixed Recent Searches to update UI when using “clear” button. ([#4923](https://github.com/brave/brave-ios/issues/4923))

### PR DESCRIPTION
**Pull request by Uni-verse**
## Summary of Changes

This pull request fixes [#8029](https://github.com/brave/brave-ios/issues/8029)

## Submitter Checklist:

- [ ] *Unit Tests* are updated to cover new or changed functionality
- [ ] User-facing strings use `NSLocalizableString()`
- [ ] New or updated UI has been tested across:
  - [ ] Light & dark mode
  - [ ] Different size classes (iPhone, landscape, iPad)
  - [ ] Different dynamic type sizes

## Test Plan:

N/A

## Reviewer Checklist:

- [ ] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [ ] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [ ] Adequate unit test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Review by iccub — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,52 @@
...
+### Web3
+
+- Added search bar to “Select a Token to Send” view. ([#7768](https://github.com/brave/brave-ios/issues/7768))
+- Added “Group by” filter to Portfolio tab in Brave Wallet. ([#7765](https://github.com/brave/brave-ios/issues/7765))
+- Added NFT filters and display settings in the NFT tab for Brave Wallet. ([#7699](https://github.com/brave/brave-ios/issues/7699))
+- Added new UI for the Brave Wallet onboarding flow. ([#7619](https://github.com/brave/brave-ios/issues/7619))
+- Added handling of empty state(s) for Brave Wallet Portfolio filters. ([#7618](https://github.com/brave/brave-ios/issues/7618))
+- Added filters and display settings for Portfolio tab in Brave Wallet. ([#7585](https://github.com/brave/brave-ios/issues/7585))
+- Enabled Sardine On-ramp purchase option for all wallet users. ([#7608](https://github.com/brave/brave-ios/issues/7608))
+- Fixed crash when switching to Polygon Network in Brave Wallet “Swap” view. ([#7907](https://github.com/brave/brave-ios/issues/7907))
```
Comment (srirambv) [inline CHANGELOG.md]: nit: Network -> network

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,52 @@
...
 
+## [1.57](https://github.com/brave/brave-ios/releases/tag/v1.57)
+
+### Web3
+
+- Added search bar to “Select a Token to Send” view. ([#7768](https://github.com/brave/brave-ios/issues/7768))
+- Added “Group by” filter to Portfolio tab in Brave Wallet. ([#7765](https://github.com/brave/brave-ios/issues/7765))
+- Added NFT filters and display settings in the NFT tab for Brave Wallet. ([#7699](https://github.com/brave/brave-ios/issues/7699))
+- Added new UI for the Brave Wallet onboarding flow. ([#7619](https://github.com/brave/brave-ios/issues/7619))
+- Added handling of empty state(s) for Brave Wallet Portfolio filters. ([#7618](https://github.com/brave/brave-ios/issues/7618))
```
Comment (srirambv) [inline CHANGELOG.md]: nit: state(s) -> state. 

No multiple states available so no need to pluralize it

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,52 @@
...
+- Updated “Block Cookie Consent Notices” to be enabled by default. ([#7882](https://github.com/brave/brave-ios/issues/7882))
+- Updated logic for the Default Browser prompt on New Tab Page. ([#7860](https://github.com/brave/brave-ios/issues/7860))
+- Updated Sync Chain authentication logic to support Biometrics when enabling password sync or adding a new device. ([#7849](https://github.com/brave/brave-ios/issues/7849))
+- Updated that VPN Support Contact Form to append the issue category to the email subject field as well as body. ([#7828](https://github.com/brave/brave-ios/issues/7828))
+- Fixed issue with “Block Cookie Consent Notices” toggle switch being enabled when upgrading browser version. ([#7920](https://github.com/brave/brave-ios/issues/7920))
+- Fixed issue with Google OAuth window not auto-closing after successfully authenticating. ([#6966](https://github.com/brave/brave-ios/issues/6966))
+- Fixed URLs for i.redd.it posts on subreddits from forming incorrectly when using [old.reddit.com](http://old.reddit.com/) website redirect. ([#7873](https://github.com/brave/brave-ios/issues/7873))
```
Comment (srirambv) [inline CHANGELOG.md]: nit:  add quotes for i.redd.it

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,52 @@
...
+- Fixed “Brave can now be set as your default browser…” prompt to not be shown 14 days after installation. ([#7977](https://github.com/brave/brave-ios/issues/7977))
+- Fixed crash when migrating with very many tabs open. ([#7857](https://github.com/brave/brave-ios/issues/7857))
+- Fixed crash when viewing an advertisement in the Brave News feed after joining Brave Rewards. ([#7837](https://github.com/brave/brave-ios/issues/7837))
+- Fixed Tabs Bar to update properly when opening bulk bookmarks. ([#7836](https://github.com/brave/brave-ios/issues/7836))
+- Fixed restored but unopened tabs from being added to “Recently Closed Tabs” page. ([#7832](https://github.com/brave/brave-ios/issues/7832))
+- Fixed tabs to close properly with window.close(). ([#7796](https://github.com/brave/brave-ios/issues/7796))
+- Fixed crash in tab view when tapping “Private” flag while actively dragging a tab. ([#7737](https://github.com/brave/brave-ios/issues/7737))
+- Fixed Recent Searches to update UI when using “clear” button. ([#4923](https://github.com/brave/brave-ios/issues/4923))
```
Comment (srirambv) [inline CHANGELOG.md]: nit: quotes for "Recent Searches" to keep it consitent

**Review by srirambv — CHANGES_REQUESTED** [review]
Rest looks good

**Comment by Uni-verse** [comment]
> Rest looks good

Thanks @srirambv, changes are made.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,52 @@
...
+- Added cookie consent notice popup as opt-out. ([#7792](https://github.com/brave/brave-ios/issues/7792))
+- Added “View Recently Closed Tabs” to the tab tray context menu. ([#7752](https://github.com/brave/brave-ios/issues/7752))
+- Added network level standard blocking for cosmetic filtering. ([#7539](https://github.com/brave/brave-ios/issues/7539))
+- Updated the strings in the “restore your tabs” message (after experiencing a crash). ([#7695](https://github.com/brave/brave-ios/issues/7695))
+- Updated the URL bar context menu to show by long pressing on the search field as well the refresh icon in the URL bar. ([#7642](https://github.com/brave/brave-ios/issues/7642))
+- Updated New Tab Page favicon to display Brave icon instead of generic planet icon. ([#5357](https://github.com/brave/brave-ios/issues/5357))
+- Updated “Block Cookie Consent Notices” to be enabled by default. ([#7882](https://github.com/brave/brave-ios/issues/7882))
+- Updated logic for the Default Browser prompt on New Tab Page. ([#7860](https://github.com/brave/brave-ios/issues/7860))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: I don't think `Default Browser` needs to be capitalized in this instance. Can just be the following:

```
- Updated logic for the default browser prompt on New Tab Page. ([#7860](https://github.com/brave/brave-ios/issues/7860))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,52 @@
...
+- Updated Sync Chain authentication logic to support Biometrics when enabling password sync or adding a new device. ([#7849](https://github.com/brave/brave-ios/issues/7849))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: Probably don't need to have `Biometrics` capitalized in this instance.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,52 @@
...
+- Updated that VPN Support Contact Form to append the issue category to the email subject field as well as body. ([#7828](https://github.com/brave/brave-ios/issues/7828))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: remove `that` from the beginning. Don't think it's needed.

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,52 @@
...
+- Fixed issue with “Block Cookie Consent Notices” toggle switch being enabled when upgrading browser version. ([#7920](https://github.com/brave/brave-ios/issues/7920))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: probably don't need `version` at the end. It's obvious that the version will be different when updating. Can just use:

```
- Fixed issue with “Block Cookie Consent Notices” toggle switch being enabled when upgrading browser. ([#7920](https://github.com/brave/brave-ios/issues/7920))
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,52 @@
...
+- Fixed issue with Google OAuth window not auto-closing after successfully authenticating. ([#6966](https://github.com/brave/brave-ios/issues/6966))
+- Fixed URLs for "i.redd.it" posts on subreddits from forming incorrectly when using [old.reddit.com](http://old.reddit.com/) website redirect. ([#7873](https://github.com/brave/brave-ios/issues/7873))
+- Fixed issue allowing users to bypass the Browser Lock window using the “Cancel” button. ([#7893](https://github.com/brave/brave-ios/issues/7893))
+- Fixed “Brave can now be set as your default browser…” prompt to not be shown 14 days after installation. ([#7977](https://github.com/brave/brave-ios/issues/7977))
+- Fixed crash when migrating with very many tabs open. ([#7857](https://github.com/brave/brave-ios/issues/7857))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: doesn't really sound right with `very`. Did the migration happen during update? Maybe we can call that out for some context. Maybe something as follows:

```
- Fixed crash when migrating after update when browser has many tabs opened. ([#7857](https://github.com/brave/brave-ios/issues/7857))
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,52 @@
...
+- Fixed crash when viewing an advertisement in the Brave News feed after joining Brave Rewards. ([#7837](https://github.com/brave/brave-ios/issues/7837))
+- Fixed Tabs Bar to update properly when opening bulk bookmarks. ([#7836](https://github.com/brave/brave-ios/issues/7836))
+- Fixed restored but unopened tabs from being added to “Recently Closed Tabs” page. ([#7832](https://github.com/brave/brave-ios/issues/7832))
+- Fixed tabs to close properly with window.close(). ([#7796](https://github.com/brave/brave-ios/issues/7796))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Can probably remove this one from the notes. I don't most people will understand the above. Maybe we can change it to something as follows:

```
Fixed tabs not being closed correctly in certain cases.
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,52 @@
...
+- Fixed crash in tab view when tapping “Private” flag while actively dragging a tab. ([#7737](https://github.com/brave/brave-ios/issues/7737))
```
Comment (kjozwiak) [inline CHANGELOG.md]: I don't have a lot of context but is `flag` the right word? Is it referring to the `Private` button?

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,5 +1,52 @@
...
+- Updated logic for the Default Browser prompt on New Tab Page. ([#7860](https://github.com/brave/brave-ios/issues/7860))
+- Updated Sync Chain authentication logic to support Biometrics when enabling password sync or adding a new device. ([#7849](https://github.com/brave/brave-ios/issues/7849))
+- Updated that VPN Support Contact Form to append the issue category to the email subject field as well as body. ([#7828](https://github.com/brave/brave-ios/issues/7828))
+- Fixed issue with “Block Cookie Consent Notices” toggle switch being enabled when upgrading browser version. ([#7920](https://github.com/brave/brave-ios/issues/7920))
+- Fixed issue with Google OAuth window not auto-closing after successfully authenticating. ([#6966](https://github.com/brave/brave-ios/issues/6966))
+- Fixed URLs for "i.redd.it" posts on subreddits from forming incorrectly when using [old.reddit.com](http://old.reddit.com/) website redirect. ([#7873](https://github.com/brave/brave-ios/issues/7873))
+- Fixed issue allowing users to bypass the Browser Lock window using the “Cancel” button. ([#7893](https://github.com/brave/brave-ios/issues/7893))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: Lets but `Browser Lock` in quotes --> `"Browser Lock"`

**Review by kjozwiak — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 7 above)
Comment (Uni-verse) [reply CHANGELOG.md]: Good catch, thanks!

File: CHANGELOG.md (hunk 10 above)
Comment (Uni-verse) [reply CHANGELOG.md]: Will use your suggestion. Thanks.

**Comment by Uni-verse** [comment]
@kjozwiak Thanks for reviewing. https://github.com/brave/brave-ios/commit/7ee8accb03510fa1d5927529186db5396846c905

**Review by kjozwiak — APPROVED** [review]
LGTM 👍

**Review by srirambv — APPROVED** [review]
++

**Pull request merged**